packages/backend/src/peripherals/database/RollupStateRepository.ts

```diff
@@ -131,7 +131,7 @@ export class RollupStateRepository implements IRollupStateStorage {
 
 function parametersToJson(parameters: RollupParameters) {
   return {
-    timestamp: parameters.timestamp.toString(),
+    timestamp: BigInt(Number(parameters.timestamp)),
```

Comment:
There are no bigints in json. This has to be a number then

Reply:
Returned type is not really a json - `timestamp` is of type `bigint` and `funding` of `jsonb`. I think naming here is confusing, so I changed `json` to `row`, which reflects reality.